cli/printer.js

```diff
@@ -58,13 +59,17 @@ function writeToStdout(output) {
  */
 function writeFile(filePath, output, outputMode) {
   return new Promise((resolve, reject) => {
-    // TODO: make this mkdir to the filePath.
-    fs.writeFile(filePath, output, (err) => {
-      if (err) {
+    fs.mkdir(path.dirname(filePath), { recursive: true }, (err) => {
```

Comment:
Can you add a test for this in `cli/test/cli/printer-test.js`

Reply:
All set 😄